L1Trigger/TrackerTFP/python/KalmanFilterFormats_cfi.py

```diff
@@ -17,52 +17,59 @@
   WidthC22 = cms.int32( 20 ),
   WidthC23 = cms.int32( 20 ),
   WidthC33 = cms.int32( 20 ),
+  
+  Widthchi2 = cms.int32( 8 ),
 
-  BaseShiftx0           = cms.int32(   0 ),
-  BaseShiftx1           = cms.int32(  -7 ),
-  BaseShiftx2           = cms.int32(  -1 ),
-  BaseShiftx3           = cms.int32(  -1 ),
+  BaseShiftx0           = cms.int32(  -2 ),
```

Comment:
Do we have any README.md explaining how to tune these parameters?

Reply:
if tuning is needed, system detects an overflow, is will throw an exception explaining how to tune it.

Reply:
I'd suggest saying this in a README somewhere.